README.md

```diff
@@ -1,19 +1,24 @@
-# GitOps Toolkit
+# Flux version 2
 
 [![e2e](https://github.com/fluxcd/toolkit/workflows/e2e/badge.svg)](https://github.com/fluxcd/toolkit/actions)
 [![report](https://goreportcard.com/badge/github.com/fluxcd/toolkit)](https://goreportcard.com/report/github.com/fluxcd/toolkit)
 [![license](https://img.shields.io/github/license/fluxcd/toolkit.svg)](https://github.com/fluxcd/toolkit/blob/main/LICENSE)
 [![release](https://img.shields.io/github/release/fluxcd/toolkit/all.svg)](https://github.com/fluxcd/toolkit/releases)
 ![overview](docs/diagrams/gitops-toolkit.png)
 
-The GitOps Toolkit is a set of composable APIs and specialized tools
-that can be used to build a Continuous Delivery platform on top of Kubernetes.
+Flux is a tool for keeping Kubernetes clusters in sync with
+configuration in Git repositories, and automating updates to
```

Comment:
Given that Flux v2 can work with S3 buckets and Helm repositories only, should we reformulate "configuration in Git repositories"?

Reply:
Yes, if there's a good replacement term -- I'm reluctant to spin sentences out further for the sake of precision. I could stretch it to "Git repositories (and similar)" but it gets a bit vague.

Reply:
Turned it around, I think it covers your point and still reads OK now.

Reply:
Great!